tests/network/probes.go

```diff
@@ -112,7 +112,7 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				}, 120)).ToNot(HaveOccurred())
 
 				// Marking the status to not ready can take a little more time
-				tests.WaitForVMICondition(virtClient, vmi, v1.VirtualMachineInstanceReady, 300)
+				tests.WaitForVMIConditionRemovedOrFalse(virtClient, vmi, v1.VirtualMachineInstanceReady, 300)
```

Comment:
Shame on me! Please mention in the commit message that you are correcting a mistake introduced in commit 11c666ba99.

Reply:
Done.